packages/myst-cli/src/project/types.ts

```diff
@@ -24,6 +24,12 @@ export type LocalProjectPage = {
   implicit?: boolean;
 };
 
+export type ExternalURL = {
```

Comment:
I think to follow the pattern of where this is slotting in, maybe it should be called `LocalProjectUrl`? (That name is a little funny on it's own, but I think these urls are present alongside `LocalProjectPage | LocalProjectFolder` on `pages` below.)

Reply:
I suspect you have a much clearer view of the overarching architecture, so I will defer to you on naming that is not confusing in the LocalProject context, and also clearly communicates that it is an external URL.

It may even be that I attached the information in the wrong place. Let me know, and I'm happy to change it.

Reply:
`LocalProjectExternalURL` feels the most explicit and consistent to me. +1 on just being a bit verbose and doing that.